src/main/java/com/stripe/param/AccountCreateParams.java

```diff
@@ -0,0 +1,2642 @@
+   * `company`.
+   */
+  @SerializedName("company")
+  Company company;
+
+  /**
+   * The country in which the account holder resides, or in which the business is legally
+   * established. This should be an ISO 3166-1 alpha-2 country code. For example, if you are in the
+   * United States and the business for which you're creating an account is legally represented in
+   * Canada, you would use `CA` as the country for the account being created.
+   */
+  @SerializedName("country")
+  String country;
+
+  /**
+   * Three-letter ISO currency code representing the default currency for the account. This must be
+   * a currency that [Stripe supports in the account's country](https://stripe.com/docs/payouts).
+   */
+  @SerializedName("default_currency")
+  String defaultCurrency;
+
+  /**
+   * The email address of the account holder. For Standard accounts, Stripe will email your user
+   * with instructions on how to set up their account. For Custom accounts, this is only to make the
+   * account easier to identify to you: Stripe will never directly email your users.
+   */
+  @SerializedName("email")
+  String email;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  /**
+   * A card or bank account to attach to the account. You can provide either a token, like the ones
+   * returned by [Stripe.js](https://stripe.com/docs/stripe.js), or a dictionary, as documented in
+   * the `external_account` parameter for [bank
+   * account](https://stripe.com/docs/api#account_create_bank_account) creation. <br>
+   * <br>
+   * By default, providing an external account sets it as the new default external account for its
+   * currency, and deletes the old default if one exists. To add additional external accounts
+   * without replacing the existing default for the currency, use the bank account or card creation
+   * API.
+   */
+  @SerializedName("external_account")
+  Object externalAccount;
+
+  @SerializedName("from_recipient")
+  String fromRecipient;
+
+  /**
+   * Information about the person represented by the account. This field is null unless
+   * `business_type` is set to `individual`.
+   */
+  @SerializedName("individual")
+  Individual individual;
+
+  /**
+   * A set of key-value pairs that you can attach to an `Account` object. This can be useful for
+   * storing additional information about the account in a structured format.
+   */
+  @SerializedName("metadata")
+  Map<String, String> metadata;
+
+  /**
+   * The set of capabilities you want to unlock for this account. Each capability will be inactive
+   * until you have provided its specific requirements and Stripe has verified them. An account may
+   * have some of its requested capabilities be active and some be inactive.
+   */
+  @SerializedName("requested_capabilities")
+  List<RequestedCapability> requestedCapabilities;
```

Comment:
I did not know we were going with enums in params. Is that expected? While I'm fine in principle, I would say that enum on resources are higher priority than on params (personal opinion). This is a bad example as it's not returned in the API but I've confirmed that `disabled_reason` for example on the `Account` is not an enum.

Reply:
Yup this is expected. Enum on resource introduced now will cause too much incompatibility with 7.x. But here since we start from zero, we introduce Enum as well.

Reply:
Ack okay. Does Java let you pass a string there easily instead of an enum value? There are a lot of properties that have gated enum values

Reply:
We have `.toMap` exposed to users so they can always add gated/beta content the Map, and use the old way of calling API